### ISSUE TITLE
Use extension name in custom media definitions?

### ISSUE THREAD

**Issue by hgl** [postcss/postcss-custom-media#9]
Do you think this should be supported?

```css
@custom-media --foo (width < 10em);
@custom-media --bar (--foo);
```

I also asked it on [www-style](https://lists.w3.org/Archives/Public/www-style/2015Apr/0213.html).

**Comment by hgl** [comment]
Got answer from csswg.

I can send a PR if you think the similar `preserve: true | "definitions"` option can be used here.

**Comment by timkelty** [comment]
Curious about this too.

**Comment by MoOx** [comment]
Closed by 590e4146b344e6583db275c6a0c2c41ebb0b364a
